lib/variable/slice.cpp

```diff
@@ -51,7 +51,7 @@ auto get_coord(const Variable &coord, const Dim dim) {
 void expect_same_unit(const Variable &coord, const Variable &value,
                       const std::string &name) {
   if (coord.unit() != value.unit()) {
-    throw except::UnitError("The unit of the " + name + " of the slice key (" +
+    throw except::UnitError("The unit of the slice " + name + " (" +
                             to_string(value.unit()) +
```

Comment:
Isn't this back to where we where? For me "slice value" refers to the value of the data array we are slicing, not the key used for selecting the slice.

Reply:
I changed how the function is called. Now it passed `"key"`.